Gemfile

```diff
@@ -0,0 +1,39 @@
+# vim:ft=ruby
+source 'https://rubygems.org'
+
+if ENV.key?('PUPPET_VERSION')
+  puppetversion = "#{ENV['PUPPET_VERSION']}"
+else
+  puppetversion = '~> 5.0'
+end
+
+group :development, :unit_tests do
+
+  gem 'json',                    '~> 2.1'
+  gem 'json_pure',               '~> 2.1'
+  gem 'CFPropertyList',          '~> 2.3'
+  gem 'metadata-json-lint',      '~> 2.0'
+  gem 'semantic_puppet',         '~> 1.0' if puppetversion =~ /4./
+  gem 'rgen',                    '~> 0.8' if puppetversion =~ /4./
```

Comment:
Assuming this is related to the requirements stuff from the website, it might make more sense to do something like this:

```ruby
if puppetversion =~ /^4\./
  # puppet 4 gems
elsif puppetversion =~ /^5\./
  # puppet 5 gems...
end
```

(you'll want to double check that regex btw)